### PR TITLE
Update project to support Skillz 12 requirements

### PR DESCRIPTION
**Pull request by osowskit**
Ability to add Embedded Frameworks
Ability to add LD Runtime Search Paths

### CODE REVIEW

File: XCProject.cs [hunk 1]
```diff
@@ -378,6 +393,22 @@ public PBXDictionary AddFile( string filePath, PBXGroup parent = null, string tr
 							string frameworkPath = Path.Combine( "$(SRCROOT)", Path.GetDirectoryName( filePath ) );
 							this.AddFrameworkSearchPaths(new PBXList(frameworkPath));
 						}
+						
+						if(embed)
+						{
+							string ldRuntimePath = "$(inherited) @executable_path/Frameworks";
+							this.AddLDRuntimeSearchPaths(ldRuntimePath);
+							//LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
```
Comment (MattFoley) [inline XCProject.cs]: Should we just drop this line?

**Comment by MattFoley** [comment]
Nothing here looks worrisome to me. The post process script would then be setup where?

File: XCProject.cs (hunk 1 above)
Comment (osowskit) [inline XCProject.cs]: Yep.  It is also possible we could remove the entire AddLDRuntimeSearchPaths method since it should work without it.

**Comment by osowskit** [comment]
This PR doesn't support adding the Runscript build phase.  Since adding a new build phase isn't as invasive as modifying an existing field, I've chosen to skip refactoring that for now.

It will continue to be handled by the existing XCode export.  Note that this project doesn't contain any Skillz specific files other than namespace.

**Comment by osowskit** [comment]
@MattFoley removed the comment. per your request

**Comment by MattFoley** [comment]
:+1:

**Pull request merged**